bin/handler-opsgenie.rb

```diff
@@ -120,7 +120,7 @@ def create_alert
 
   def tags
     tags = []
-    tags << json_config['tags'] if json_config['tags']
+    tags += json_config['tags'] if json_config['tags']
```

Comment:
This is quite the subtle "bug" rather than concatenating the two arrays into a new array it adds the object which happens to be an array into the original array which creates a multidimensional array.
```irb
$ irb
irb(main):001:0> tags = []
irb(main):003:0> tags << [] if true
=> [[]]
irb(main):004:0> tags = []
=> []
irb(main):005:0> tags += []
=> []
```
I double checked the array libs and confirmed:
- https://docs.ruby-lang.org/en/2.4.0/Array.html#method-i-2B
- https://docs.ruby-lang.org/en/2.4.0/Array.html#method-i-3C-3C